packages/google_maps_flutter/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: google_maps_flutter
 description: A Flutter plugin for integrating Google Maps in iOS and Android applications.
 homepage: https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter
-version: 0.5.22+3
+version: 0.5.22+4
```

Comment:
Because this patch adds a new feature to the plugin, we need to bump the minor version. Hence 0.5.23.
https://dart.dev/tools/pub/versioning#semantic-versions